Dockerfile-Base

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:16.04
+LABEL maintainer = "david.thor@architect.io"
+
+# Install Node.js & Python
+RUN apt-get update && \
+  apt-get -y upgrade && \
+  apt-get install -y build-essential autoconf libtool pkg-config curl git sudo unzip python3-pip && \
+  pip3 install --upgrade pip click grpcio && \
+  curl -sL https://deb.nodesource.com/setup_10.x -o nodesource_setup.sh && \
+  bash ./nodesource_setup.sh && \
+  apt-get install -y nodejs
+
+# Map python3 to python
+RUN sudo rm /usr/bin/python && \
```

Comment:
Why does sudo need to be installed and run here? When you're running a container you should already be root by default